glib/Gir.toml

```diff
@@ -232,7 +232,16 @@ status = "generate"
     manual = true
     [[object.function]]
     name = "mkstemp"
-    #manual is_windows_utf8
+    #manual is_windows_utf8 and also modifies the string passed in
```

Comment:
`#manual`?

Reply:
It's manually implemented. This is the same comment as in the 50 lines above and IIRC you added that back then ;)